qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -279,14 +285,15 @@ def observable(matrix_elements, init_term=0, mapping="jordan_wigner", wires=None
 
     **Example**
 
-    >>> t = np.array([[0., 0., 0.5], [1.0, 1.0, -0.5], [1.0, 0., 0.]])
-    >>> v = np.array([[ 0., 0., 0., 0., 0.25], [ 0., 1., 1., 0., -0.25], [ 1., 0., 0., 1., -0.5]])
-    >>> matrix_elements = [t, v]
-    >>> print(observable(matrix_elements, init_term=1/4, mapping="bravyi_kitaev"))
-    (0.15625) [I0]
-    + (-0.15625) [Z0]
-    + (0.34375) [Z0 Z1]
-    + (-0.09375) [Z1]
+    >>> t = FermionOperator("0^ 0", 0.5) + FermionOperator("1^ 1", 0.25)
+    >>> v = FermionOperator("1^ 0^ 0 1", -0.15) + FermionOperator("2^ 0^ 2 0", 0.3)
+    >>> print(observable([t, v], mapping="jordan_wigner"))
```

Comment:
Any reason why init_term is removed from the example?

Reply:
``init_term`` now is less important than before since the contribution of core orbitals is handled internally. You will use it only to pass a constant for shifting the expectation value of the observable. That's why I prefer to keep the example as simple as possible.